Automattic/wc-calypso-bridge
# Folder structure for new release automation script

### Changes proposed in this Pull Request:


This sets up the initial folder structure and dependencies for where we're going to start creating a release automation script.

Related to #1096






### How to test the changes in this Pull Request:




1. Check out this branch.
2. In the `./scripts` directory, run `yarn` to install dependencies.
3. In the root of the project run `yarn run create-release`
4. You should see a message reading "Coming Soon".



### Other information:




### FOR PR REVIEWER ONLY:
